chia/data_layer/data_layer_wallet.py

```diff
@@ -167,7 +167,7 @@ async def create_new_dl_wallet(
         await self.standard_wallet.push_transaction(regular_record)
         await self.standard_wallet.push_transaction(dl_record)
         await self.wallet_state_manager.update_wallet_puzzle_hashes(self.wallet_info.id)
-        return self
+        return ItemAndSpendBundleNames(item=self, spend_bundle_names={spend_bundle.name()})
```

Comment:
i suggest we return the whole tx record like in the pool wallet, not sure what the advantage of creating a new type as opposed to using Tuple like in the pool wallet